Sources/NIO/BSDSocketAPI.swift

```diff
@@ -308,6 +308,9 @@ extension NIOBSDSocket.Option {
     /// IPv4 and IPv6.
     static let ip_recv_tos: NIOBSDSocket.Option =
             NIOBSDSocket.Option(rawValue: IP_RECVTOS)
+
+    static let ip_recv_pktinfo: NIOBSDSocket.Option =
```

Comment:
Can you add a doc comment to this in both places? It's only for us, but I'd like to address what it's for and note why it's `private` (namely, that we want users to use the `ChannelOption`).